geo/src/algorithm/contains.rs

```diff
@@ -404,18 +404,19 @@ mod test {
     fn point_polygon_with_inner_test() {
         let linestring = LineString::from(vec![(0., 0.), (2., 0.), (2., 2.), (0., 2.), (0., 0.)]);
         let inner_linestring = LineString::from(vec![
-            (0.5, 0.5),
-            (1.5, 0.5),
-            (1.5, 1.5),
-            (0.0, 1.5),
-            (0.0, 0.0),
+            [0.5, 0.5],
+            [1.5, 0.5],
+            [1.5, 1.5],
+            [0.0, 1.5],
+            [0.0, 0.0],
         ]);
         let poly = Polygon::new(linestring, vec![inner_linestring]);
-        assert!(poly.contains(&Point::new(0.25, 0.25)));
+        assert!(!poly.contains(&Point::new(0.25, 0.25)));
```

Comment:
Confusingly, as a result of closing the ring, this went from `true` to `false` 🤔